drivers/nutdrv_qx_hunnox.c

```diff
@@ -0,0 +1,137 @@
+/* nutdrv_qx_zinto.c - Subdriver for Zinto protocol based UPSes
```

Comment:
@marianojan : if this was adapted from a subdriver for another UPS model, worth mentioning taht but the title here should point to Hunnox ;)